src/common/components/BaniList/baniOrderHelper.js

```diff
@@ -0,0 +1,23 @@
+const oderedBani = (baniList) => {
+  const defaultBaniOrder = require("../../defaultBaniOrder.json");
+  const banis = [];
+
+  defaultBaniOrder.baniOrder.forEach((obj) => {
+    if (obj.id) {
+      const baniItem = baniList.filter((item) => item.id === obj.id)[0];
+      if (baniItem) {
+        banis.push({ id: obj.id, gurmukhi: baniItem.gurmukhi, translit: baniItem.translit });
+      }
+    } else {
+      const folder = [];
+      obj.folder.forEach((item) => {
+        const bani = baniList.filter((listItem) => listItem.id === item.id)[0];
+        folder.push({ id: item.id, gurmukhi: bani.gurmukhi, translit: bani.translit });
+      });
+
+      banis.push({ gurmukhi: obj.gurmukhi, translit: obj.translit, folder });
+    }
```

Comment:
You can write this in a function, so the code will be a bit self-explanatory. Maybe name something like `createFolderBani` and pass the required variables in arguments.

Likewise you can create function for the above if statement and name that something like `createSingleBani`

Reply:
fixed. I tried to simplify the code not this way but other way.